src/main/java/mateAcademy/EntityManagerHolder.java

```diff
@@ -0,0 +1,22 @@
+package mateAcademy;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+public class EntityManagerHolder {
+    protected static EntityManager em;
+
+    static {
+        EntityManagerFactory emf = Persistence
+                .createEntityManagerFactory("PatientPU");
+        em = emf.createEntityManager();
+    }
+
+    public static EntityManager getEntityManager() {
+        return em;
+    }
+
+    private EntityManagerHolder() {
+    }
+}
```

Comment:
An extra line should be in your file

Reply:
And Why have you written method private EntityManagerHolder() {
    }? Why do you need it here ?

Reply:
@boroda4436 Sorry, yes I agree that this class must be a Singleton, But constructor is a special method in class, or group of instructions that initialize an object. It depends on definition